includes/class-sensei-grading-main.php

```diff
@@ -472,7 +531,29 @@ public function get_views() {
 				break;
 		endswitch;
 
-		$counts = Sensei()->grading->count_statuses( apply_filters( 'sensei_grading_count_statues', $count_args ) );
+		/**
+		 * Filter count statuses arguments in Grading.
+		 *
+		 * @hook sensei_grading_count_statues
+		 *
+		 * @deprecated $$next-version$$ Contains typo. Use sensei_grading_count_statuses.
+		 *
+		 * @param {array} $count_args Count statuses arguments.
+		 * @return {array} Filtered count arguments.
+		 */
+		$count_args = apply_filters( 'sensei_grading_count_statues', $count_args );
```

Comment:
Should we use the [apply_filters_deprecated](https://developer.wordpress.org/reference/functions/apply_filters_deprecated/) instead?

Reply:
Makes sense 👍